docs/marionette.application.md

```diff
@@ -2,21 +2,34 @@
 
 # Marionette.Application
 
-The `Backbone.Marionette.Application` object is the hub of your composite
-application. It organizes, initializes and coordinates the various pieces of your
-app. It also provides a starting point for you to call into from your HTML
-script block, or directly from your JavaScript files if you prefer to go that
-route.
+The **Application** is a container for the rest of your code. It is recommended
+that every Marionette app have at least one instance of Application.
 
-The `Application` is meant to be instantiated directly, although you can extend
-it to add your own functionality.
+By creating an Application you get three important things:
 
-```js
-var MyApp = new Backbone.Marionette.Application();
-```
+- A `start` method to kick off your application.
+  This allows you an opportunity to do things that may need to occur before, say, you
+  begin routing. An example would be making an AJAX call to request data that your app
+  needs before starting.
+
+- A namespace to keep things off of the `window`.
+  If you are not using a module loader like ES6 modules, CommonJS, or AMD, then
+  you can use the Application to store your Javascript objects. And if you are
+  using one of those module systems, then you can still attach things to the
+  application to aid in debugging.
+
+- Integration with the Marionette Inspector. The Marionette Inspector is a fantastic tool
+  that makes it easy to understand and debug your application. Using the Application Class
+  will automatically hook up your application to that extension.
```

Comment:
nope - just needs to exist. And honestly, I'd like to remove the `app` dependency entirely. So, I'm not sure I really need this paragraph